packages/terra-status-view/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Added
```

Comment:
I think this should be `### Fixed` since this is a bug fix. Also, might be beneficial to add updated style to use display block for inline bug fix (or something with more context)

Reply:
https://github.com/cerner/terra-core/pull/2218/commits/05004c807b09dd8f7d0dfcf9744a33030f53f3ff